app/card-info/[id]/page.tsx

```diff
@@ -0,0 +1,20 @@
+// The detailed information of a card if just a whole seperate page with one card by itself
+// It is a solution made on the basis of non existent requirements for this page
+// So a developer went on the path of a least resistance
+"use client";
+import { useParams } from "next/navigation";
+import { useTodo } from "../../../context/TodoContext";
+import TodoCard from "../../components/TodoCard";
+
+export default function CardDetailsPage() {
+  const { cardData } = useTodo();
+  const params = useParams();
+  const { id } = params;
+  const idNumber = +id;
```

Comment:
what is the reason for line 13?

Reply:
It is for comparison, to get card info on the line 14. ID from URL is string that must be converted to number before comparison can happen.